.github/PULL_REQUEST_TEMPLATE/New_Feature.md

```diff
@@ -0,0 +1,24 @@
+---
+name: 🎉 New Feature
+about: Have you implemented some neat idea that you want to make part of arkitect? 🎩
+---
+
+<!--
+Thank you for submitting new feature!
+-->
+
+### New Feature
+
+<!-- Fill in the relevant information below to help triage your issue. -->
+
+|    Q        |   A
+|------------ | ------
+| New Feature | yes
+| RFC         | yes/no
```

Comment:
Sorry, I forgot to say that RFC is here too.. a part from that it's great for me :+1: